lib/ui/index.js

```diff
@@ -355,6 +355,7 @@ class UI {
             this.log(debugInfo, 'yellow');
             const logLocation = system.writeErrorLog(stripAnsi(`${debugInfo}\n${output}`));
             this.log(`\nAdditional log info available in: ${logLocation}`);
+            this.log(`\nTry running ${(chalk.cyan('ghost doctor'))} to check your system for known issues.`)
```

Comment:
Hm shouldn't this be logged for all three cases? I saw lot's of people getting `An unknown error occurred`.

Reply:
Yeah. You're right... 🙈 Changes will come in a sec.